app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,47 @@
+class GroupsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_group, only: [:show, :edit, :update, :destroy]
```

Comment:
I tend to prefer using assign methods like this:

```ruby
def group
  @group ||= Group.find(params[:id])
end
```

and then use the `group` method in the actions. Do you prefer that? I can easily switch it up. But it doesn't really solve the issue of the source of `@group` and still hide it somewhere... maybe I can put it back in the actions, but that's a lot of duplicated code (for easier understanding). What do you think @ptrikutam?

I'm pretty open about the style and can adapt depending on who I'm working with, so feel free to tell me your favorite approach and I will use it for every project :)